plugins/org.eclipse.sirius.debug/src/org/eclipse/sirius/api/debug/TabularReport.java

```diff
@@ -37,7 +38,7 @@ public void addLine(List<String> data) {
     }
 
     public void addLine(String... data) {
```

Comment:
Checkstyle rule: method to comment (use same comment for above method)
```suggestion
    /**
     * Add one line for each {@link String} of data list.
     * 
     * @param data
     *            The list of lines to add.
     */
    public void addLine(String... data) {
```

Reply:
done